src/pages/lessons/fundamentals/eth-intro-part-3.mdx

```diff
@@ -0,0 +1,209 @@
+# sneks everywhere
+```
+
+## How far can a contract go?
+
+So long as we're talking about the management of digital assets, you can program nearly anything. Physical assets have been tokenized on the blockchain too, but that's another rabbit hole.
+
+Over the years, standards for various digital assets have proposed, debated, and agreed upon, providing some foundational building blocks for more complex contracts. Among the most notable are the ERC-20 token standard (i.e., fungible tokens) and the ERC-721 standard (i.e., non-fungible tokens or "NFTs").
+
+<Callout emoji="💡" size="md" variant="info">
+**Note**: To save you the Google search, _fungible_ means interchangeable or indistinguishable. In other words, if you own 100 fungible tokens, it doesn't make any difference which 100 tokens they are. NFTs, on the other hand, may each have unique qualities, so the particular token you own is significant.
+</Callout>
+
+To demystify those standards: the different token types are simply smart contract patterns that anyone can make use of. The ERC-20 standard specifies which functions your fungible token contract must include, but at its core, the contract simply maintains list of public addresses and how many tokens each one owns, represented by an integer.
+
+The ERC-721 standard overlaps with ERC-20, but importantly introduces a unique token ID and some metadata for each token.
+
+These two standards are brought up to illustrate their compounding effect. As more of these building blocks are standardized, the easier it gets to quickly stack them in creative ways and innovate at the fringes.
+
+## Contracts within contracts
+
+Continuing the comparison to object-oriented classes, _inheritance_ is another concept you will commonly find in contracts. Well-trusted organizations like OpenZeppelin implement token standard contracts, for example, so that contract developers can simply import and inherit that functionality.
+
+Within Solidity, inheritance is declared using the `is` keyword:
+
+```solidity
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MyToken is ERC20 {
+    constructor() ERC20("ExampleToken", "XMPL") { ... }
+}
+```
+
+Once deployed, the `MyToken` contract has access to all the functions defined in OpenZeppelin's `ERC20` contract implementation. Conveniently, you don't have to reinvent the wheel and can focus on what makes your token contract unique.
+
+Beyond inheritance, contracts have the ability to interact with other deployed contracts or even serve as a _factory_ or _proxy_ for deploying still more contracts! Those concepts are good topics for future blog posts.
+
+## A note on upgradeability
+
+While the blockchain is said to be immutable, there are patterns of writing smart contracts that can support upgradeability. These patterns introduce trade-offs which may or may not make sense for your use case. The specifics are beyond the scope of this introductory lesson.
+
+## Starting off on your own
```

Comment:
Generally speaking, there are two sorts of links here. Help resources, and learning resources. 

Before we started Academy, we tasked ourselves with creating two Notion tracks for  web2 and web3, and two Discord channels to manage them proactively. It was these sorts of rescources we were compiling. We eventually abandoned it, and started building School of Code => Academy. 

I don't think anyone wants to deny advice to a learner to good quality content mentioned here, but at the same time, `we're abandoning the mission and vision of this education platform`, and in my mind, what our DAO has potentially to offer.

Culture building is really important. Raza, or DappaDan, or any other potential authors might just look in at this, and think 'oh links are okay, or an easy way' out of writing their own self-supporting material. And then our platform is saturated with links, and when we do get our own Ethereum Stack Exchange up and running, moderators are going to be tasked with trying to answer questions far removed from their own work. 

Starting humbly can also be an attractive aspect for budding authors. They'll know that the attention-factor on Academy will be all theirs.

I'd say a better approach would be to leave a couple of help links e.g. Solidity by Example and Remix, and pursue a more homegrown approach suggesting an invite inwards: https://handbook.developerdao.com/

Reply:
i altogether nuked this section and links within, encouraging continuation within Academy. agree on the importance of simple focus at this stage in the learners journey.